website/project/metadata/egap-registration.json

```diff
@@ -0,0 +1,418 @@
+					"help": "E.g., 06/02/2018"
+				},
+				{
+					"qid": "q12",
+					"title": "B8 Should this study be gated? (discouraged)",
+					"nav": "Should this study be gated?",
+					"type": "string",
+					"format": "text",
+					"description": "If Yes, indicate release date (studies can be gated for up to 18 months or the date of publication, if earlier). We discourage gating but sometimes this is necessary to protect subjects or the integrity of the research. If the study should not be gated, please LEAVE THIS FIELD BLANK.",
+					"help": "E.g., 06/02/2018"
+				},
+				{
+					"qid": "q13",
+					"title": "B9 Was this design presented at an EGAP meeting?",
+					"nav": "Presented at an EGAP meeting?",
+					"type": "choose",
+					"format": "singleselect",
+					"options": [
+						"N/A",
+						"No",
+						"Yes"
+					],
+					"description": "Indicate if the design received feedback from a EGAP design workshop or other special EGAP session prior to registration"
+				},
+				{
+					"qid": "q14",
+					"title": "B10 Is there a pre-analysis plan associated with this registration?",
+					"nav": "Pre-analysis plan associated with this registration?",
+					"type": "choose",
+					"format": "singleselect",
+					"options": [
+						"N/A",
+						"No",
+						"Yes"
+					],
+					"description": "If so, please attach it in the Additional Documentation section below"
+				}
+			]
+		},
+		{
+			"id": "page2",
+			"title": "Registration Data",
+			"questions": [{
+					"qid": "q14",
+					"nav": "Background and explanation of rationale.",
+					"title": "C1 Background and explanation of rationale.",
+					"format": "textarea",
+					"required": true,
+					"description": "Brief description of goals of project. If you are also attaching a pre-analysis plan, please refrain from simply copying and pasting a section from your plan here. If possible, please also avoid saying \"see attached pre-analysis plan,\" as it renders the search functionality less useful. Rather, please provide a short (1-2 paragraph) summary of the project background."
+				},
+				{
+					"qid": "q15",
+					"nav": "Text format",
+					"title": "Text format",
+					"type": "choose",
+					"format": "singleselect",
+					"options": [
+						"Plain text",
+						"Filtered test"
+					],
+					"help": "Plain Text:<ul><li>Use [fn]...[/fn] (or <fn>...</fn>) to insert automatically numbered footnotes.</li><li>No HTML tags allowed. </li><li>Web page addresses and e-mail addresses turn into links automatically.</li><li>Lines and paragraphs break automatically.</li></ul>Filter Text:<ul><li>Use [fn]...[/fn] (or <fn>...</fn>) to insert automatically numbered footnotes.</li><li>Allowed HTML tags: &lt;a&gt; &lt;em&gt; &lt;strong&gt; &lt;cite&gt; &lt;blockquote&gt; &lt;code&gt; &lt;ul&gt; &lt;ol&gt; &lt;li&gt; &lt;dl&gt; &lt;dt&gt; &lt;dd&gt; </li><li>Lines and paragraphs break automatically.</li></ul>"
```

Comment:
what is this? same w/ q17, q19

Reply:
If you go to upload a document, this is a question that they ask, are you going to copy/paste with plain text or filtered text.

Reply:
have since removed and have instructions about plain text.  I'll remove these form blocks.